pkgs/development/libraries/libowlevelzs/default.nix

```diff
@@ -0,0 +1,41 @@
+{ cmake,
+  fetchurl,
+  lib,
+  stdenv,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libowlevelzs";
+  version = "0.0.3";
+
+  src = fetchurl {
+    url = "https://ytrizja.de/distfiles/${pname}-${version}.tar.gz";
+    sha256 = "49892b9ccd0943c23dc5af842f929ce9467ea50abd6a0e985f506464e8e8bfe0";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  postPatch = ''
+    patchShebangs config_relink.sh
+    sed -i -e "s/cstdint/stdint.h/g" src/hash.hpp
+    sed -i -e "s/std::uintmax_t/uintmax_t/g" src/hash.hpp
+  '';
+
+  preBuild = ''
+    cmakeFlagsArray+=(
+      "-DBUILD_SHARED_LIBS=true"
+      "-DINSTALL_LIB_DIR=$out/lib"
+      "-DINSTALL_INCLUDE_DIR=$out/include/zs/ll"
+    )
+  '';
+
+  enableParallelBuilding = true;
```

Comment:
Not required: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unnecessary-parallel-building.md